build.gradle

```diff
@@ -325,15 +325,10 @@ dependencies {
 
     implementation group: 'io.github.openfeign', name: 'feign-httpclient', version: '12.4'
     implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.11.0'
-    implementation group: 'com.squareup.okio', name: 'okio-jvm', version: '3.5.0'
 
     implementation group: 'com.github.hmcts', name: 'auth-checker-lib', version: '2.1.5'
     implementation group: 'com.github.hmcts', name: 'properties-volume-spring-boot-starter', version: '0.1.1'
-
-    implementation group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: '2.0.2'
-
-
-    implementation group: 'com.github.hmcts', name: 'java-logging', version: '6.0.1'
+    implementation group: 'com.github.hmcts', name: 'java-logging', version: '5.1.9'
```

Comment:
keep that at 6.0.1 and add this dependency
`    implementation group: 'net.logstash.logback', name: 'logstash-logback-encoder', version: '7.4'`

Reply:
locally it fails if we use 6.0.1 @benouaer

Reply:
sorry, forgot to mention that you have to remove those interceptors that need that dependency
see https://github.com/hmcts/sscs-tribunals-case-api/pull/3478